pages/wiki/ble-profiles.md

```diff
@@ -54,7 +54,7 @@ layout: documentation
   <h1 id="notificationprofile">Notification Profile</h1>
 </div>
 
-<p>Clients can create notifications on the watch side by using the Notification Profile. This profile is typically used to synchronize notifications from a phone to a watch or to send special alerts such as AsteroidOSSync's "Find my watch" feature.</p>
+<p>Clients can create notifications on the watch side by using the Notification Profile. This profile follows the <a href="https://developer.apple.com/library/archive/documentation/CoreBluetooth/Reference/AppleNotificationCenterServiceSpecification/Specification/Specification.html">Apple Notification Center Service (ANCS) Specification</a>. This profile is typically used to synchronize notifications from a phone to a watch or to send special alerts such as AsteroidOSSync's "Find my watch" feature.</p>
```

Comment:
That's not correct.

We use our own custom Notification BLE profile on all platforms *except* for iOS where we use ANCS. It would be more appropriate to add a totally separate section where we explain that for iOS support specifically, we have the watch side of ANCS implemented.